### ISSUE TITLE
Appear on current screen on multi screen setup

### ISSUE THREAD

**Issue by jcaonr** [kvakulo/Switcheroo#162]
Hi Regin, thanks for this nice little tool.

Issue: I have a four monitor setup and always looking around to find Switcheroo. 
Idea: Is it somehow possible to open the SwitcherooUI on the current screen (where the mouse is)? 

Thanks  a lot for your effort and sry if I missed an option to do it for myself.

**Comment by hiyorijl** [comment]
came here to write same issue, please it would be sooo useful 🙏😊, in multiple screens setup switcheroo only appears on the main screen regardless of where was i clicked before / active window. 

it always appears only the first monitor and not on my second one.

**Comment by FaizanYounasTanooli** [comment]
i am working on it you will get it soon

**Comment by FaizanYounasTanooli** [comment]
https://github.com/FaizanYounasTanooli/Switcheroo

**Comment by FaizanYounasTanooli** [comment]
I am also working on its UI to make it 100% same as orignal windows Alt+Tab look

**Comment by SatoshiSolution** [comment]
Hey [FaizanYounasTanooli](https://github.com/FaizanYounasTanooli), 

thank you for your awesome work! helps me a lot!
An OPTION for displaying the Switcheroo bar on every open display would be great. Maybe that is not too much work?!

Thanks! :)

**Comment by FaizanYounasTanooli** [comment]
i am already done with it and created pull request. I have the exe if you want i have not uploaded it on my forked repo

**Comment by FaizanYounasTanooli** [comment]
oh wait i get you now 
you want it to appear on every screen?
then what you want further should every screen's switcheroo display its own screen's windows only right?
and other thing when its opened on pressing tab in which screen's window should selected ?
i guess you will be using only mouse to select window?

**Comment by SatoshiSolution** [comment]
Hey there, 
thanks for your response, and sorry for the bad communication. 

I actually thought of a switcheroo selection window appearing on both screens simultaneously, as visible in the sketch:
![image](https://github.com/kvakulo/Switcheroo/assets/122448284/12fb33ac-4683-4f60-87c8-1042cc958a10)

That would help me not having to move my head and eyes towards the main screen A, but instantly start switching from either screen A or screen B. I hope that is better understandable. 

Best

**Comment by ajasja** [comment]
@FaizanYounasTanooli Did you ever merge the PR for the switcheroo appearing on the active monitor (either the one that has the mouse or the one that has the active window)? That would be super awsome

**Comment by dvygolov** [comment]
You are welcome, guys. Implemented that behavior in my fork: https://github.com/dvygolov/Switcheroo/

**Comment by kbilsted** [comment]
> You are welcome, guys. Implemented that behavior in my fork: https://github.com/dvygolov/Switcheroo/

nice. do you do binary releases somewhere ? are you taking over the project?

**Comment by dvygolov** [comment]
Yep, created a release.yml with necessary build actions, now the releases
are built automatically when new updates are pushed into master. I don't
think that I'll "take over" the project, but if you have any good ideas to
implement - feel free to add an issue, I'll think about it. But right now I
think that Switcheroo is almost feature complete and doesn't need anything
else.

**Comment by coezbek** [comment]
I merged the nice patch from @dvygolov into my fork at https://github.com/coezbek/switcheroo including also the Github release action for building releases from main.
